controllers/actions.summerwind.net/autoscaling_test.go

```diff
@@ -534,8 +584,9 @@ func TestDetermineDesiredReplicas_OrganizationalRunner(t *testing.T) {
 			workflowRuns:             `{"total_count": 4, "workflow_runs":[{"status":"in_progress"}, {"status":"in_progress"}, {"status":"in_progress"}, {"status":"completed"}]}"`,
 			workflowRuns_queued:      `{"total_count": 0, "workflow_runs":[]}"`,
 			workflowRuns_in_progress: `{"total_count": 3, "workflow_runs":[{"status":"in_progress"},{"status":"in_progress"},{"status":"in_progress"}]}"`,
-			want:                     3,
+			want:                     1,
```

Comment:
Good catch! However, I think this makes this test case useless- How about adding `"id":123` or whatever to every `workflow_runs` items so that it still falls into the desired code path, allowing this `want` value to remain `3`?

Reply:
@mumoshu, looked a bit through the code but I don't think it makes any difference.
The fixed value influences the replicas of the `RunnerDeployment` and that ultimately is the `scaleTarget` but to calculate the desired amount of replicas needed, none of that information is used.
The workflows don't have a `self-hosted` label, which would result in the requirement of 3. Otherwise they're treated as 3 hosted runners and that means the desired replicas == minimum defined as they're not of importance.

All autoscaler tests are running against the `AutoscalingMetricTypeTotalNumberOfQueuedAndInProgressWorkflowRuns` metric and none against `AutoscalingMetricTypePercentageRunnersBusy`. For the later, I could imagine the `fixed`, and therefore `scaleTarget` to have an influence but since we only check for `inQueue / inProgress` the `fixed` value and, therefore, replicas of the RunnerDeployment have 0 influence on the `desired` value.

Reply:
@mumoshu, I'd propose removing the `fixed` tests, since they're irrelevant for `AutoscalingMetricTypeTotalNumberOfQueuedAndInProgressWorkflowRuns` tests.
Are you in accord with the change?

Reply:
@mumoshu , just another ping since this is stalling a bit ...

Reply:
@mumoshu another ping, we would really love this to get merged and released. This change is very important as it will have a positive impact on the cost of running multiple clusters

Reply:
@nikola-jokic @toast-gear hey guys, any chance you can provide some feedback here too and help to get this merged? this is a very important change and we will all benefit from it

Reply:
Hey, sorry for the delay,

I can't help much here since I am responsible for the `actions.github.com` API group. However, I asked @mumoshu to do another round of reviews and he is the best one to talk to related to the `actions.summerwind.net` group :relaxed: 

While waiting for the review, @Langleu do you mind updating the branch? That can fix rest of the tests. I can do it myself if you agree but it would be better if you do it

Reply:
Thank you!

Reply:
lint looks like a timeout, works locally.
Same for the RunnerSet related tests since this change doesn't touch that logic and runs into timeout as well.

Reply:
No worries, I'll re-run the tests. I know it is not affecting that part of the code

Reply:
Thanks for your assistance @nikola-jokic!